src/styles/stars.scss

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background: url('../images/star.svg') no-repeat;
+    background-position: center;
+    width: $starsSize;
```

Comment:
The variable `$starsSize` is used but not defined. Ensure it is defined in your SCSS files to avoid compilation errors.